src/GraphQl/Action/EntrypointAction.php

```diff
@@ -94,21 +95,17 @@ private function parseRequest(Request $request): array
         }
 
         if ('json' === $request->getContentType()) {
-            $input = json_decode($request->getContent(), true);
-
-            if (isset($input['query'])) {
-                $query = $input['query'];
-            }
+            return $this->parseInput($query, $variables, $operation, $request->getContent());
+        }
 
-            if (isset($input['variables'])) {
-                $variables = \is_array($input['variables']) ? $input['variables'] : json_decode($input['variables'], true);
-            }
+        if (false !== mb_stripos($request->headers->get('CONTENT_TYPE'), 'multipart/form-data')) {
+            if ($request->request->has('operations')) {
```

Comment:
What if there is no `operations`?

Reply:
Then query and variables will be null and in here:
https://github.com/api-platform/core/blob/master/src/GraphQl/Action/EntrypointAction.php#L67 we will return `GraphQL query is not valid` error.